_pages/ato/background.md

```diff
@@ -1,16 +1,45 @@
 ---
-title: Background on the ATO Process
+title: Background on the Risk Management Process
 navtitle: Background
 ---
 
-***[This page is in need of a rewrite.](https://github.com/18F/before-you-ship/issues/89)***
+One of the goals of the Federal Information Security Management Act of 2002 (FISMA) is to "provide a comprehensive framework for ensuring the effectiveness of information security controls over information resources that support Federal operations and assets." NIST was tasked with designing and implementing this framework: the result is NIST's Risk Management Framework (RMF). All federal information and information systems (except classified information and national security systems) are subject to NIST's RMF by law.
```

Comment:
My general mode is regardless of context, each standalone document always spells out acronyms the first time within that document. Plz spell out NIST - thx!